src/ontology/mondo-edit.obo

```diff
@@ -62177,15 +62177,13 @@ id: MONDO:0003406
 name: sleep-wake disorder
 def: "Abnormal sleep-wake schedule or pattern associated with the circadian rhythm which affect the length, timing, and/or rigidity of the sleep-wake cycle relative to the day-night cycle." [MESH:D012893]
```

Comment:
@twhetzel The definition mentions that the sleep-wake disruption is associated with the circadian rhythm. What makes it different from "circadian rhythm sleep disorder"?
I don't see an explanation or what made you think that these terms were not the same concepts and therefore they should both be maintained.

Reply:
The reason was in the [initial PR comment](https://github.com/monarch-initiative/mondo/pull/8119#issue-2491092342).

Reply:
The current definition in MESH seems different than what you have above: https://www.ncbi.nlm.nih.gov/medgen/?term=D012893

Reply:
Hopefully my Pending comments from yesterday have come through. The first sentence is from MeSH, but I modified the second sentence since we do not yet have the other sleep wake grouping terms that the MeSH definition mentions, but tried to add enough information with examples of existing Mondo terms to help differentiate the definition of 'sleep-wake disorder' from 'circadian rhythm sleep disorder'. Let me know what's the best way to finalize this definition.

Reply:
ah, sounds good. Could you just add your ORCID to the def then, since you modified it? Then I think this PR is good to go

Thanks for all your work on this!

Reply:
ORCID added